### ISSUE TITLE
Can I set BN trainable?

### ISSUE THREAD

**Issue by maozezhong** [jwyang/faster-rcnn.pytorch#234]
Hello, every one , I am a new pytorcher. I want to set BN trainable in pytorch. So I comment some code in `faster-rcnn.pytorch/lib/model/faster_rcnn/resnet.py  line 266 267 283 284` , Is that correct?
In my opinion, if I want to train my own datasets and my data distribution is not like COCO or VOC, so I need to set BN layer trainable during my training.
@jwyang , Am I correct? thanks advance~

**Comment by maozezhong** [comment]
Another question, I find `model.eval` in pytorch means disable some layer like BN or dropout. So why do you `set_bn_fix` when` _init_modules` and in the same time `set_bn_eval` in train function, in my opinion, they do the same work, right? @jwyang 
correct me if I am wrong , thanks~
